### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by ziirish**
- add a HOWTO easily migrate from Flask-RESPlus (ref #99)
- explain about the fork in the doc

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -4,19 +4,36 @@
...
+Flask-RESTX is a community driven fork of `Flask-RESTPlus
+<https://github.com/noirbizarre/flask-restplus>`_
+
+
+Why did we fork?
+================
+
+The community has decided to fork the project due to lack of response from the
+original author @noirbizarre. We have been discussing this eventuality for
+almost `a long time <https://github.com/noirbizarre/flask-restplus/issues/593>`_.
```
Comment (SteadBytes) [inline doc/index.rst:24]: > almost a long time

"almost" doesn't quite make sense here :smile:

File: doc/quickstart.rst [hunk 2]
```diff
@@ -10,6 +10,28 @@ and that you have already installed both Flask and Flask-RESTX.
 If not, then follow the steps in the :ref:`installation` section.
 
 
+Migrate from Flask-RESTPlus
+---------------------------
+
+.. warning:: The *migration* commands provided bellow are here for the illustration.
```
Comment (SteadBytes) [inline doc/quickstart.rst:16]: Small typo: `bellow -> below`

> for the illustration

I think "for illustration purposes" is the common form of this phrase :smile:

File: doc/quickstart.rst [hunk 3]
```diff
@@ -10,6 +10,28 @@ and that you have already installed both Flask and Flask-RESTX.
...
+This can be done using something like:
+
+::
+   find . -type f -name "*.py" | xargs sed -i "s/flask_restplus/flask_restx/g"
+
+Finally, you will need to update your configuration options (described `here
+<quickstart.html#configuration>`_). Example:
+
+::
+   find . -type f -name "*.py" | xargs sed -i "s/RESTPLUS_/RESTX_/g"
```
Comment (SteadBytes) [inline doc/quickstart.rst:32]: Nice :+1:  I always love some Linux CLI magic :smiley:

**Comment by ziirish** [comment]
Thanks for the review!

I'll resolve the comment ASAP.

File: doc/quickstart.rst (hunk 2 above)
Comment (ziirish) [reply doc/quickstart.rst]: Thanks, I rephrased this note according to your suggestion

**Comment by SteadBytes** [comment]
@ziirish I think you've hit all my review comments, are you happy with this now? If so we can merge 😊

**Comment by ziirish** [comment]
Yes, we're good to go ;)

**Pull request merged**